autobuild/windows/autobuild_windowsinstaller_3_copy_files.ps1

```diff
@@ -33,11 +34,24 @@ if (($jamulus_buildversionstring -eq $null) -or ($jamulus_buildversionstring -eq
 
 # Rename the file
 echo "rename"
-$artifact_deploy_filename = "jamulus_${Env:jamulus_buildversionstring}_win.exe"
-echo "rename deploy file to $artifact_deploy_filename"
-cp "$jamulus_project_path\deploy\Jamulus*installer-win.exe" "$jamulus_project_path\deploy\$artifact_deploy_filename"
-
+if ($BuildOption -Eq "jackonwindows")
```

Comment:
If `BuildOption` were `jack`, rather than `jackonwindows`, you could set the deploy filename like
`..._${BuildOption}_win.exe`  And then the `cp` below would be `installer-${BuildOption}-win.exe`...

Reply:
I know, but that is not the buildoption name. There also is a jackonmac option, so believe it will be best to keep it like this to avoid confusion.

Reply:
Ah, right.  Thought it was probably something like that.  Maybe one set up at the top to create the insert value (empty by default, then set to "jack" if jack, and then the insert value is inserted appropriately if not empty)?  I'm thinking if we have PortAudio as well, that could then just be one more value that is recognised once by the script, without needing to modify everywhere all over again.

Reply:
Okay, I would propose the following in the autobuild-copy-files.ps1 file:

Add the beginning: When a BuildOption is provided and this option is "jackonwindows", set the BuildOption to "jack"

Now there will be two options to copy/rename the final articfact name:

1. No BuildOption set, handle as currently and don't alter the installer name.
2. When BuildOption is set, include the BuildOption in the installer name as part of the "artifact_deploy_filename"

This should allow the flexibility to create different output artifact names, depending on the option specified.

Please let me know if you agree with this approach, so I can update the script and PR.

Reply:
I've had a quick look through for anything else and added comments.  But the idea is that _any_ part that can be factored out and used the same way _should be_ factored out.  So it's not just the deploy artifact name: there are a few other bits.

Reply:
BuildOption is now use when renaming the generated installer. The installer name remains generic until the artifact is deploy/uploaded.